src/components/Label/Label.tsx

```diff
@@ -4,11 +4,18 @@ import cx from 'classnames';
 type LabelProps = {
   className?: string;
   htmlFor: string;
+  isDisabled?: boolean;
```

Comment:
`disabled` is being used on everything else in this PR. Is there a specific reason you're using `isDisabled` here?

Reply:
I noticed that we are using `disabled` as a prop for [button](https://github.com/wellcometrust/corporate-components/blob/develop/src/components/Button/Button.tsx#L16) and [checkboxes](https://github.com/wellcometrust/funding-platform-frontend/blob/develop/src/components/CheckboxGroup/CheckboxGroup.jsx#L15), so for the tags that have a disabled attribute.

Label doesn't have a disabled attribute so I thought the way to differentiate it is to use `isDisabled` as it only adds a class.

Now when I am looking at the input component, we have [isRequired](https://github.com/wellcometrust/corporate-components/blob/develop/src/components/TextInput/TextInput.tsx#L9), but maybe it should be `required`??

Reply:
That logic makes sense, I thought that might be the case.

> Now when I am looking at the input component, we have isRequired prop, but maybe it should be required??

I agree, if you've taken that approach with `disabled` you should follow the same pattern for `required`

Reply:
ok, I will create a separate ticket to update it (I will have to make changes in CR too)